### PR TITLE
ARC-2101 add restart checkbox

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Add a checkbox to force full backfill.
image

**Why**
Per the issue to help customer with per project base server to cloud migration.

**Added feature flags**
N/A

**Affected issues**  
ARC-2101

**How has this been tested?**  
Unit test
Local

**Whats Next?**
N/A

### CODE REVIEW

File: src/routes/jira/sync/jira-sync-post.test.ts [hunk 1]
```diff
@@ -150,4 +150,31 @@ describe("sync", () => {
 			});
 	});
 
+	it("should run full sync if explicit selected by user", async () => {
```
Comment (rachellerathbone) [inline src/routes/jira/sync/jira-sync-post.test.ts:153]: nit: explicit -> explicitly

File: static/css/jira-configuration-new.css [hunk 2]
```diff
@@ -113,6 +113,10 @@
     margin-top: 1em;
 }
 
+#jiraConfiguration__restartBackfillModal__form .jiraConfiguration__restartBackfillModal__fullsync_label {
+  display: inline;
```
Comment (rachellerathbone) [inline static/css/jira-configuration-new.css:117]: Can we add 'cursor: pointer' here?

File: views/jira-configuration-new.hbs [hunk 3]
```diff
@@ -156,6 +156,18 @@
...
                   />
                 </div>
+                {{#if isIncrementalBackfillEnabled }}
+                <div class="field-group">
+                  <input class="jiraConfiguration__restartBackfillModal__fullsync"
+                         type="checkbox"
+                         id="backfill-fullsync-checkbox"
+                         name="backfill-fullsync-checkbox"
+                  />
+                  <label for="backfill-fullsync-checkbox" class="jiraConfiguration__restartBackfillModal__fullsync_label">
```
Comment (rachellerathbone) [inline views/jira-configuration-new.hbs:166]: nit: `_label` -> `__label`

File: static/css/jira-configuration-new.css (hunk 2 above)
Comment (gxueatlassian) [reply static/css/jira-configuration-new.css:117]: done.

File: src/routes/jira/sync/jira-sync-post.test.ts (hunk 1 above)
Comment (gxueatlassian) [reply src/routes/jira/sync/jira-sync-post.test.ts]: done.

File: views/jira-configuration-new.hbs (hunk 3 above)
Comment (gxueatlassian) [reply views/jira-configuration-new.hbs]: done.

**Review by rachellerathbone — APPROVED** [review]

**Pull request merged**